pygeoapi/openapi.py

```diff
@@ -68,7 +65,7 @@
 THISDIR = os.path.dirname(os.path.realpath(__file__))
 
 
-def get_ogc_schemas_location(server_config: dict) -> str:
+def get_ogc_schemas_location(server_config) -> dict:
```

Comment:
This function's type annotation got mixed up and is now incorrect - it seemed to be correct before

Reply:
Fixed